packages/web-app-files/src/App.vue

```diff
@@ -104,7 +104,6 @@ export default {
 main {
   height: 100%;
   max-height: 100%;
-  overflow: hidden;
```

Comment:
I checked and it was not needed anymore with this. Flo changed the overflow in `web-runtime/src/layouts/Application.vue` so that the `.app-container` has the overflow instead of the `.app-content`, that fixed it.